config.md

```diff
@@ -308,7 +308,10 @@ The semantics are the same as `Path`, `Args` and `Env` in [golang Cmd](https://g
 
 This OPTIONAL property contains arbitrary metadata for the container.
 This information MAY be structured or unstructured.
-Annotations are key-value maps.
+Annotations MUST be key-value maps.
+Keys MUST be unique, and best practice is to namespace the keys.
```

Comment:
Should we give an example and mention that "opencontainers.org" namespace is reserved for use in the spec?